### DISCUSSION TITLE
Templates functionality

### DISCUSSION THREAD

**Discussion by andrzejkupczyk** [andrzejkupczyk/mantisbt-todolists#47] [Ideas]
Allow to define default lists (e.g. per bug category) and append it automatically to a newly created bug report.

**Comment by lgnutek** [comment]
This function would be perfect option. 

We're using MantisBT in our company mostly for managing process flow and we usually have the same "to-do-list" in every issue. With possibility to define default to-do list attached to every reported issue it would make this small but extremely useful plug-in for us.

Any chance for udpate in this matter?

**Reply by andrzejkupczyk** [reply]
The further development of this plugin is strictly dependent on people's interest in using it. If this feature is desired, it is definitely worth adding 👍🏻
